.github/workflows/deploy-docs.yml

```diff
@@ -0,0 +1,41 @@
+name: Deploy Docs to GitHub Pages
+
+on:
+  release:
```

Comment:
I would index and deploy at the same time (on commit?)

Maybe these two workflows can be combined into a single workflow with two jobs.